src/commands.rs

```diff
@@ -3236,16 +3273,11 @@ fn rebase_to_other_parent(
     if own.parent_ids() == other.parent_ids() {
         Ok(own.tree())
     } else {
-        let own_parent_tree = merge_commit_trees(
-            workspace_command.repo().as_repo_ref(),
-            &other.parents()
-        );
-        let other_parent_tree = merge_commit_trees(
-            workspace_command.repo().as_repo_ref(),
-            &own.parents(),
-        );
-        let rebased_tree_id =
-            merge_trees(&own_parent_tree, &other_parent_tree, &own.tree())?;
+        let own_parent_tree =
+            merge_commit_trees(workspace_command.repo().as_repo_ref(), &other.parents());
+        let other_parent_tree =
+            merge_commit_trees(workspace_command.repo().as_repo_ref(), &own.parents());
+        let rebased_tree_id = merge_trees(&own_parent_tree, &other_parent_tree, &own.tree())?;
```

Comment:
This looks like a reformatting that should go in the parent commit (you can use `jj squash -i` to do that)